custom_components/alexa_media/sensor.py

```diff
@@ -210,7 +210,7 @@ def _trigger_event(self, time_date) -> None:
         )
         self.hass.bus.async_fire(
             "alexa_media_notification_event",
-            event_data={"email": hide_email(self._account), "event": self._active[0]},
+            event_data={"email": hide_email(self._account), "device": {"name": self._name}, "event": self._active[0]},
```

Comment:
Ideally it would be the entity ID of the sensor and/or media player. But basically my use case would be to be able to receive the event and then check some attributes on the correct _next_timer sensor within the automation. At the moment the event data doesn't contain anything which I can map to an entity.

Reply:
Thanks. I just added both name and entity_id.